INPUT_SCHEMA.json

```diff
@@ -102,16 +102,25 @@
         "saveSubsToKVS": {
             "title" : "Save downloaded subtitles to Key Value Store",
             "type": "boolean",
-            "description" : "If set to true, we will save downloaded (and converted to .srt format) subtitles to Key Value Store in format { subtitles: srt_text }. 'downloadSubtitles' must be set to true for this take an effect"
+            "description" : "If set to true, we will save downloaded (and converted to .srt format) subtitles to Key Value Store in format { subtitles: srt_text, language: languageCode, type:autoOrUserGenerated }. 'downloadSubtitles' must be set to true for this take an effect"
         },
         "subtitlesLanguage": {
-            "title" : "Download subtitles in selected language",
+            "title" : "Download subtitles in selected language.",
             "type": "string",
-            "description" : "Language to download subtitles in. 'downloadSubtitles' must be set to true for this take an effect",
-            "editor": "select",
-            "default" : "en",
-            "enum" : ["en", "de", "es", "fr", "it", "ja", "ko", "nl", "pt", "ru"],
-            "enumTitles" : ["English", "German", "Spanish", "French", "Italian", "Japanese", "Korean", "Holland", "Portuguese", "Russian"]
+            "description" : "Language to download subtitles in, using standardized language codes (eg. 'en', 'de', 'ru' and so on). 'downloadSubtitles' must be set to true for this take an effect. If set to `null` or empty string, we download all available subtitles.",
+            "default" : null,
+            "editor": "textfield"
+        },
+        "preferAutoGeneratedSubtitles": {
+            "title": "Prefer automatically generated subtitles before the user generated ones.",
+            "description": "If set to `true` and language is provided, we download automatically generated subtitles rather then user ones. If no language is provided, this settings takes no effect",
+            "type": "boolean"
+        },
+        "scrapeCommentCount": {
+            "title" : "Attempt to scrape N comments from video detail page.",
+            "type" : "integer",
+            "description" : "Try to scrape N comments from video detail page. -1 ~ Don't scrape any, 0 ~ Scrape all available comments, 1-x ~ scrape (1-x) comments.",
```

Comment:
1. I would not use `try` and `N`. We can simply say something like `"Limit the number of comments that will get scraped. Scraping comments requires scrolling and takes time"`

2. I really don't like this -1 and 0 system. I know it is common in some programmatic cases and we used it elsewhere but replaced it with much more logical system.
a) set "`minimum"` to `0` (negative number is not possible)
b) `0` means 0 comments (default)
c) if you want all comments, simply put `99999`